src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,64 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private final StorageImpl<K,V>[] list = new StorageImpl[MAX_ITEMS_NUMBER];
+    private K typeK;
```

Comment:
Move this field to the inner class. Fix in all places
```suggestion
    private K key;
```